app/messaging/application/process-application.js

```diff
@@ -10,8 +10,9 @@ const validateApplication = require('../schema/process-application-schema')
 const processApplication = async (msg) => {
   const { sessionId } = msg
   const applicationData = msg.body
+  const messageId = msg.messageId
   let existingApplicationReference = null
-  console.log('Application received:', util.inspect(applicationData, false, null, true))
+  console.log(`Application received : ${util.inspect(JSON.stringify(applicationData), false, null, true)} with sessionID ${sessionId} and messageID ${messageId}.`)
```

Comment:
can we remove the `util.inspect` part as I noticed this does weird things in app inights

Reply:
@liamwilson-kainos like this?

`console.log(`Application received : ${JSON.stringify(applicationData)} with sessionID ${sessionId} and messageID ${messageId}.`)`

Reply:
👍